Sources/GoogleAI/PartsRepresentable.swift

```diff
@@ -77,3 +78,42 @@ extension [any PartsRepresentable]: PartsRepresentable {
     }
   }
 #endif
+
+extension CGImage: PartsRepresentable {
+  public var partsValue: [ModelContent.Part] {
+    let output = NSMutableData()
+    guard let imageDestination = CGImageDestinationCreateWithData(
+      output, UTType.jpeg.identifier as CFString, 1, nil
+    ) else {
+      Logging.default.error("[GoogleGenerativeAI] Couldn't create JPEG from CGImage.")
+      return []
+    }
+    CGImageDestinationAddImage(imageDestination, self, nil)
+    CGImageDestinationSetProperties(imageDestination, [
+      kCGImageDestinationLossyCompressionQuality: 0.8,
```

Comment:
There has been some discussion around which compression quality to pick. To make sure they all get changed to the same value (if ever changed), would you mind extracting the 4 instances of `0.8` into a JPEG compression quality constant?

Reply:
Done.